plans/external.pp

```diff
@@ -0,0 +1,43 @@
+# @summary
+#   A plan that generates external facts based on the provided modulepath and
+#   sets facts on specified targets.
+#
+# @param path The absolute path to the directory on localhost containing external facts
+# @param targets The targest the collect and set facts on
+# @return The target objects, with facts added
+# @example Gather external facts from an installed module
+#   $moduledir = module_directory('mymod')
+#   $with_facts = run_plan(facts::external, $targets, path => file::join($moduledir, 'facts.d'))
+#   return $with_facts.map |$target| { $target.facts }
+plan facts::external(
+  String $path,
+  TargetSpec $targets
+) {
+  $localhost = get_target('localhost')
```

Comment:
What is this for?

Reply:
Whoops, this is leftover from before I added the `dir::entries` function. Removed!